WifiManager/impl/WifiManagerConnect.cpp

```diff
@@ -29,7 +29,16 @@ using namespace WPEFramework::Plugin;
 
 uint32_t WifiManagerConnect::connect(const JsonObject &parameters, JsonObject &response)
 {
-    LOGINFOMETHOD();
+    if (params.HasLabel("passphrase"))
```

Comment:
Did u mean parameters here instead of params?We need to pass the actual passphrase to the underlying layer and just suppress the logging. Pls verify that it builds and works.

Reply:
Forgot to add one line.
Added commit to fix this.